ghcide/test/exe/Main.hs

```diff
@@ -855,7 +855,18 @@ watchedFilesTests = testGroup "watched files"
 
 insertImportTests :: TestTree
 insertImportTests = testGroup "insert import"
-  [ checkImport "above comment at top of module" "CommentAtTop.hs" "CommentAtTop.expected.hs" "import Data.Monoid"
+  [ (`xfail` "known broken") (checkImport "module where keyword lower in file no exports" "WhereKeywordLowerInFileNoExports.hs" "WhereKeywordLowerInFileNoExports.expected.hs" "import Data.Int")
```

Comment:
please could you use the `expectFailBecause` existing function, if possible with a short insight about why do you think it fail ("known broken" would be fine though)

Reply:
Sure, I'll do that later today.